docs/patterns/labels.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+---
+
+## Labels
+
+<hr>
+
+Labels are static elements which you can apply to HTML elements to signify status, tags or any other information you find useful.
+
+<div class="p-notification--information">
+  <p class="p-notification__response">
+    <span class="p-notification__status">Note:</span>These labels are used to inform <a href="/component-status" class="p-notification__action">status</a> of components in Vanilla.
+  </p>
+</div>
+
+### New
+
+Label to be used on newly released components, utilities or settings that are safe to use in projects.
+
+<a href="/examples/patterns/labels/new/"
+    class="js-example">
+View example of the labels pattern
+</a>
+
+### Deprecated
+
+Label to be used on components, utilities or settings are in the process of being removed and should no longer be used in projects.
+
+<a href="/examples/patterns/labels/deprecated/"
+    class="js-example">
+View example of the labels pattern
+</a>
+
+### In progress
+
+Label to be used when a design spec and code implementation are not yet finished.
+
+<a href="/examples/patterns/labels/in-progress/"
+    class="js-example">
+View example of the labels pattern
```

Comment:
Can the change to the name of the pattern for a11y

Reply:
Fixed 👍